### PR TITLE
Chore/client setup

### PR DESCRIPTION
**Pull request by lacyjpr**

### CODE REVIEW

**Comment by lacyjpr** [comment]
Closes #24

**Comment by matty22** [comment]
While trying to install dependencies, I get an error:

```
Matts-MBP:client Matty$ yarn install
yarn install v1.7.0
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error eslint@5.0.1: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by lacyjpr** [comment]
That ruins the fun! What version of node are you on?

**Comment by matty22** [comment]
```
Matts-MacBook-Pro:~ Matty$ node -v
v9.4.0
Matts-MacBook-Pro:~ Matty$ npm -v
6.0.0
```
I'll try updating Node and see if that helps.

**Comment by lacyjpr** [comment]
I don't think it's your node...I'm on 8 something.

**Comment by matty22** [comment]
Updating node worked for me. The error said it was expecting a node version in this range

`Expected version "^6.14.0 || ^8.10.0 || >=9.10.0"`

I was running 9.5.0, so I was outside the range. Updated to 10.5.0 and it stops throwing up on that error.

**Comment by matty22** [comment]
Got the branch to build successfully after several adjustments on my machine. The PR looks good to me.

**Review by matty22 — APPROVED** [review]
Looks good to me

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "client",
```
Comment (Kornil) [inline client/package.json:2]: what about setting up `devEngines` since it already caused confusion in this PR

File: package.json [hunk 2]
```diff
@@ -1,22 +1,29 @@
 {
-  "name": "Media",
+  "name": "media",
```
Comment (Kornil) [inline package.json:2]: `devEngines`

**Review by Kornil — COMMENTED** [review]
Please run `npm audit` (make sure npm >= 5.10) and let me know the result, I have a funky feeling about the sass setup.

**Pull request merged**